documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -2333,6 +2333,58 @@ to the same shared resource is running.
 include::{testDir}/example/SharedResourcesDemo.java[tags=user_guide]
 ----
 
+==== Relevant properties
```

Comment:
Please move this content to the `Configuration` section.

Reply:
Done!
![Screenshot from 2022-06-17 13-07-14](https://user-images.githubusercontent.com/23459549/174286942-f9c90c49-8dae-4b27-88c8-2718d1bf87ad.png)